Formula/libpanel.rb

```diff
@@ -0,0 +1,41 @@
+class Libpanel < Formula
+  desc "Dock/panel library for GTK 4"
+  homepage "https://gitlab.gnome.org/GNOME/libpanel"
+  url "https://download.gnome.org/sources/libpanel/1.2/libpanel-1.2.0.tar.xz"
+  sha256 "d9055bbbab9625f3f5ce6d1fd7132eb6ea34a6ba07a87e9938901fb8b31581e2"
+  license "LGPL-3.0-or-later"
+
+  depends_on "gettext" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "vala" => :build
+  depends_on "gi-docgen"
+  depends_on "gtk4"
+  depends_on "libadwaita"
+
+  def install
+    system "meson", "setup", "build", "-Ddocs=disabled", *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <libpanel.h>
+
+      int main(int argc, char *argv[]) {
+        uint major = panel_get_major_version();
+        return 0;
+      }
+    EOS
+    flags = shell_output("#{Formula["pkg-config"].opt_bin}/pkg-config --cflags --libs libpanel-1").strip.split
+    flags += shell_output("#{Formula["pkg-config"].opt_bin}/pkg-config --cflags --libs libadwaita-1").strip.split
+    system ENV.cc, "test.c", "-o", "test", *flags
```

Comment:
Does the `--help` do anything?

Reply:
Do nothing, just copy from other formula, just keep for respect.

Reply:
Please don't add useless flags

Reply:
Got it, thanks.